src/styles/blocks/calendar.scss

```
@@ -0,0 +1,48 @@
+.calendar {
+  $root: &;
+
+  display: flex;
+  flex-wrap: wrap;
+  padding: 30px 0 30px 10px;
+  width: (7 * $cell-size) + ($cell-gap * 6) + ($calendar-padding *2);
+  border: 1px solid #000;
+  gap: $cell-gap;
+
+  @each $day, $date in $days {
+    &--start-day--#{$day} :first-child {
+      margin-left: ($cell-size + $cell-gap) * $date;
+    }
+  }
+
+  @for $lastDate from 28 through 31 {
+    &--month-length--#{$lastDate} {
+      #{$root}__cell:nth-child(n + #{$lastDate + 1}) {
+        display: none;
+      }
+    }
+  }
+
+  &__cell {
+    display: flex;
+    height: $cell-size;
+    width: $cell-size;
+    box-sizing: border-box;
+    border: 1px solid #000;
+    justify-content: center;
+    align-items: center;
+    transition-duration: 0.5s;
```

you should specify which properties you apply the transition to. in this case all is used which is not quite ok
```suggestion
    transition: background-color 0.5s, transform 0.5s;
```